### DISCUSSION TITLE
perform a wiping dalvik ?????

### DISCUSSION THREAD

**Discussion by Henk4542** [community/community#136416] [Programming Help]
### Body

Hi
I have question , I want to install something from the konstakang site.
I want to install what is written below

1 Download [http://KonstaKANG-rpi-resize.zip](http://disq.us/url?url=http%3A%2F%2FKonstaKANG-rpi-resize.zip%3A9cHuzv-8_wNFwYeXOyUXGKJWvnI&cuid=4806155) (md5:31a1d26e80cf4544a1fcd689e8799d7b)
and save it to your device’s internal storage or use an external USB drive
2 Boot to TWRP recovery (see FAQ)
3 Install [http://KonstaKANG-rpi-resize.zip](http://disq.us/url?url=http%3A%2F%2FKonstaKANG-rpi-resize.zip%3A9cHuzv-8_wNFwYeXOyUXGKJWvnI&cuid=4806155) from your selected storage
4 Boot out of recovery (see FAQ)

now I read as it says on the konstakang site, and here above , How to perform
Now I have seen ( in ) video,s that they after step 3 , do perform a wiping dalvik,

[Lineage OS 21 Android 14 on Raspberry Pi 5. Install the… — disq.us](https://disq.us/url?url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DIRFNbLIfFu0%3AMNb2bPmhmdQFPXJtc5ujoGU4800&cuid=4806155) 

on the konstakang site , they don't talk about that  (  perform a wiping dalvik after installing a zip file , ( step 3 ) 
now I am confused
do i have to perform after step 3 , a wiping dalvik,
please help
do i have to perform a wiping dalvik, can anyone tell me if you need to perform a wiping dalvik after installing a zip file , ( step 3 )

Thanks for your time and please help

![Info](https://github.com/user-attachments/assets/cd8d7945-1656-4e74-899e-e9ece7f00a9a)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Tianming-Wu** [comment]
It is not necessary since dalvik works as a cache. However, it is always suggested to wipe dalvik to prevent potential issues.

Also, wiping dalvik won't lost any data. It will be rebuilt after next startup, so it natually makes the next startup takes a little bit longer.

This zip just resize the userdata partition, it has nothing to do with the internal file, so you don't need to do that. It's a good habit though.
